cni-plugin/main.go

```diff
@@ -188,6 +207,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 			logrus.Errorf("linkerd-cni client err in client.Pods().Get(): %e", err)
 			return err
 		}
+		setupDebugLog(pod)
```

Comment:
I think it's still helpful to log stuff to a file past this point; most of the failure points occur from here on out. How I wish it'd be simpler to do it earlier though... feels like we can mix some valuable context such as configuration options, errors received from improper kubeconfig, API Server errors when fetching the pod... I'm fine personally with what we'd be missing out on, since we at least have more visibility than before, and it makes it easier to debug, but perhaps this should only be an interim fix until we find a more robust configuration method.